app/controllers/ontologies_controller.rb

```diff
@@ -219,6 +219,37 @@ def sparql
     end
   end
 
+  def content_finder
+    if params[:acronym] && params[:uri]
+      @acronym = params[:acronym]
+      @format = params[:output_format]
+      case params[:output_format]
+      when 'json'
+        accept_header = "application/json"
+      when 'xml'
+        accept_header = "application/xml"
+      when 'ntriples'
+        accept_header = "application/n-triples"
+      when 'turtle'
+        accept_header = "text/turtle"
+      end
+
+      url = URI.parse("#{rest_url}/ontologies/#{params[:acronym].strip}/resolve/#{helpers.escape(params[:uri].strip)}")
```

Comment:
create a helper to this in another file ,
this file is already too big and do a lot of things.